gensim/models/keyedvectors.py

```diff
@@ -185,95 +183,97 @@
     _load_word2vec_format,
     ft_ngram_hashes,
 )
-from gensim.similarities.termsim import TermSimilarityIndex, SparseTermSimilarityMatrix
 
-#
 # For backwards compatibility, see https://github.com/RaRe-Technologies/gensim/issues/2201
 #
 from gensim.models.deprecated.keyedvectors import EuclideanKeyedVectors  # noqa
 
 logger = logging.getLogger(__name__)
 
 
-class Vocab(object):
-    """A single vocabulary item, used internally for collecting per-word frequency/sampling info,
-    and for constructing binary trees (incl. both word leaves and inner nodes).
+class KeyedVectors(utils.SaveLoad):
+    """Abstract base class / interface for various types of word vectors."""
+    """Class containing common methods for operations over word vectors."""
+    """Mapping between words and vectors for the :class:`~gensim.models.Word2Vec` model.
+    Used to perform operations on the vectors such as vector lookup, distance, similarity etc.
 
     """
-    def __init__(self, **kwargs):
-        self.count = 0
-        self.__dict__.update(kwargs)
-
-    def __lt__(self, other):  # used for sorting in a priority queue
-        return self.count < other.count
-
-    def __str__(self):
-        vals = ['%s:%r' % (key, self.__dict__[key]) for key in sorted(self.__dict__) if not key.startswith('_')]
-        return "%s(%s)" % (self.__class__.__name__, ', '.join(vals))
-
-
-class BaseKeyedVectors(utils.SaveLoad):
-    """Abstract base class / interface for various types of word vectors."""
     def __init__(self, vector_size):
-        self.vectors = zeros((0, vector_size))
+        self.vectors = zeros((0, vector_size))  # was syn0, once upon a time
+        self.vectors_norm = None  # was syn0norm, once upon a time
         self.vocab = {}
         self.vector_size = vector_size
-        self.index2entity = []
+        self.index2key = []  # fka index2entity or index2word
```

Comment:
```suggestion
        self.index2key = []  # formerly known as index2entity or index2word
```

…I assume? Never heard that one before, let's expand at least the first instance.